app/src/main/res/layout/activity_player_queue_control.xml

```diff
@@ -31,17 +31,20 @@
         android:id="@+id/play_queue"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_above="@id/center"
+        android:layout_above="@id/progress_bar"
         android:layout_below="@id/appbar"
         android:scrollbars="vertical"
         app:layoutManager="LinearLayoutManager"
         tools:listitem="@layout/play_queue_item" />
 
+
+
     <RelativeLayout
```

Comment:
Please reorder the `RelativeLayout`s in the code according to their visual order

Reply:
> Please reorder the `RelativeLayout`s in the code according to their visual order

Sure, I check this later.